pkg/apis/extensions/v1beta1/types.go

```diff
@@ -70,66 +69,66 @@ type SubresourceReference struct {
 	Subresource string `json:"subresource,omitempty"`
 }
 
-// ResourceConsumption is an object for specifying average resource consumption of a particular resource.
-type ResourceConsumption struct {
-	// Resource specifies either the name of the target resource when present in the spec, or the name of the observed resource when present in the status.
-	Resource v1.ResourceName `json:"resource,omitempty"`
-	// Quantity specifies either the target average consumption of the resource when present in the spec, or the observed average consumption when present in the status.
-	Quantity resource.Quantity `json:"quantity,omitempty"`
+type CPUTargetUtilization struct {
+	// fraction of the requested CPU that should be utilized/used,
+	// e.g. 70 means that 70% of the requested CPU should be in use.
```

Comment:
nit: clarify the sampling frequency here. Is this x% over the last minute, or hour or day?